apps/google-analytics-4/frontend/src/services/api.ts

```diff
@@ -31,9 +31,9 @@ type Headers = Record<string, string>;
 
 export type Credentials = z.infer<typeof ZCredentials>;
 
-export class ApiError extends Error { }
-export class ApiServerError extends ApiError { }
-export class ApiClientError extends ApiError { }
+export class ApiError extends Error {}
+export class ApiServerError extends ApiError {}
+export class ApiClientError extends ApiError {}
```

Comment:
more prettier formatting noise – will be addressed holistically for ga4 app in: https://github.com/contentful/apps/pull/2653